frontend/src/scenes/onboarding/billing/FreeTierLimits.tsx

```diff
@@ -0,0 +1,79 @@
+import * as Icons from '@posthog/icons'
+
+import { availableOnboardingProducts } from '../onboardingLogic'
+
+type FreeTierLimit = {
+    title: string
+    value: string
+    icon: string
+    color: string
+}
+
+const freeTierLimits: FreeTierLimit[] = [
```

Comment:
This should all be loaded in dynamically so if it changes we don't need to upgrade in other locations. You can use the billing response for this.